apps/speed.c

```diff
@@ -2719,6 +2767,36 @@ int speed_main(int argc, char **argv)
         }
     }
 
+#ifndef OPENSSL_NO_CMAC
+    if (doit[D_EVP_CMAC]) {
+        if (evp_cmac_cipher != NULL) {
+            const char *cipher_name = OBJ_nid2ln(EVP_CIPHER_type(evp_cmac_cipher));
```

Comment:
I've no feeling either way here.

Reply:
okay, normally it won't make a difference.